src/SWP/Bundle/WebRendererBundle/Controller/DefaultController.php

```diff
@@ -34,9 +35,13 @@ public function indexAction()
             throw $this->createNotFoundException('No homepage configured!');
         }
 
-        $response = $this->render('index.html.twig', [
-            'page' => $homepage,
-        ]);
+        try {
+            $response = $this->render('index.html.twig', [
```

Comment:
Nope. This is not what we want. If there is no file then normal exception should be raised (as it was before). We should react earlier (on place where we resolve theme name from tenant and fetch theme - there we should check if theme files exists and if not then raise custom exception).